### PR TITLE
fix function arg specs 

### PR DESCRIPTION
**Pull request by flyingmachine**
see #42

### CODE REVIEW

**Comment by flyingmachine** [comment]
I'm a dope and didn't realize #43 exists, but there are some slight differences between this PR and that one, and I prefer this one

File: src/reifyhealth/specmonstah/core.cljc [hunk 1]
```diff
@@ -675,7 +676,8 @@
                               #{ref-ent} ref-ent)}))))
 
 (s/fdef ent-relations
-  :args (s/tuple ::db ::ent-name)
+  :args (s/or :n-1 (s/cat :db ::db)
+              :n-2 (s/cat :db ::db :ent-name ::ent-name))
```
Comment (codonnell-reify) [inline src/reifyhealth/specmonstah/core.cljc]: I think there shouldn't be a 2-arity spec here, since `ent-relations` always has two params.

**Comment by flyingmachine** [comment]
@codonnell-reify updated, and in record time! I also discovered that you can use `s/?` for multi-arity function specs, which is pretty cool (https://groups.google.com/forum/#!topic/clojure/jYCRLBTZNnE)

**Comment by codonnell-reify** [comment]
Did not know about `s/?` either. Thanks!

**Pull request merged**